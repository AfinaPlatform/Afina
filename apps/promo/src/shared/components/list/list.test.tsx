import List from "./list";
import { render } from "@testing-library/react";

describe("shared/list", () => {
  it("render ul", () => {
    const { container } = render(
      <List>
        <List.Item>item1</List.Item>
      </List>
    );

    expect(container.querySelector("ul")?.tagName).toBe("UL");
  });

  it("snapshot", () => {
    const { asFragment } = render(
      <List>
        <List.Item>item1</List.Item>
      </List>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
