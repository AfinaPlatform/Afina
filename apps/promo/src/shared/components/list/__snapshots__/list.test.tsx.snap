// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared/list snapshot 1`] = `
<DocumentFragment>
  <ul
    class="list"
    data-list="true"
  >
    <li
      class="item"
      data-list-item="true"
    >
      item1
    </li>
  </ul>
</DocumentFragment>
`;
